md/sv2-workshop.md

```diff
@@ -216,7 +221,11 @@ assuming `$TP` is the path to `bitcoind`
 
 ```
 cd bitcoin-sv2
-$TP -datadir=$HOME/.bitcoin-sv2-workshop -signet -sv2 -sv2port=8442
+$TP -datadir=$HOME/.bitcoin-sv2-workshop -signet -sv2
+```
+⚠️ If you grabbed a binary from releases and you're on macOS, you would probably have it in your downloads dir, so do this:
+```
+~/Downloads/Bitcoin-Qt.app/Contents/MacOS/Bitcoin-Qt -signet -sv2
```

Comment:
tbh I'm not sure about using Bitcoin-Qt. There's many steps based on `bitcoin-cli` for generating address for the coinbase, and I didn't translate them into Bitcoin-Qt because honestly I never used it.

we need to either add steps for Bitcoin-Qt (which maybe will make the slide sequence more confusing), or just use this release, where I generated the macOS binaries manually:

https://github.com/plebhash/bitcoin/releases/tag/btc-prague

Reply:
This is just for the ones who are using a Intel-based macOS

Reply:
> I didn't translate them into Bitcoin-Qt because honestly I never used it.

It's the same, minus `bitcoin-cli -datadir ...`, the Console is just an interface to the RPC.

Reply:
are you sure?

![image](https://github.com/plebhash/sv2-workshop/assets/147345153/e3524198-0684-49fa-bb60-ef0d40080883)